dffml/plugins.py

```diff
@@ -110,7 +115,8 @@ def check_boost():
         "swig": lambda: inpath("swig"),
         "cython": lambda: inpath("cython"),
     }
-    if not python_package_installed("auto_sklearn")
+    if platform.system() not in {"Windows", "Darwin"}
+    and not python_package_installed("auto_sklearn")
```

Comment:
```suggestion
    and not python_package_installed("autosklearn")
```

Package name has no `_`